config/initializers/doorkeeper.rb

```diff
@@ -18,6 +18,8 @@ def self.optional
 
   realm "Panoptes"
 
+  grant_flows ["authorization_code", "client_credentials", "implicit", "password"]
```

Comment:
They removed `implicit` and `password` from the defaults in 2.0.

* `implicit` is probably what we're using in PFE
* `password` is used by some frontends (get access token using username/password without ever redirecting to Panoptes). They shouldn't use this, but I'm not sure if they've migrated away yet

They added the other two as new defaults:

* `client_credentials` is for that Faraday middleware I wrote (get access token using application key/secret)
* `authorization_code` is the full redirect and temporary token -> access token exchange that all external apps that have a backend should be using.